tests/virt_control_plane_test.go

```diff
@@ -256,6 +256,10 @@ var _ = Describe("[Serial][ref_id:2717][sig-compute]KubeVirt control plane resil
 			}
 
 			It("should fail health checks when connectivity is lost, and recover when connectivity is regained", func() {
+				daemonSet, err := virtCli.AppsV1().DaemonSets(flags.KubeVirtInstallNamespace).Get(context.Background(), componentName, metav1.GetOptions{})
```

Comment:
This call is repeated in `readyFunc`, maybe we could refactor?

Reply:
I am not sure as I would then need to change the `Eventually`. I think 1 more call is not a problem.

Reply:
No need to change the `Eventually`, you could create a separate function that returns the daemonset (checking also for the potential error), and call it from `readyFunc` and from the body of the `It`. Just a nit in any case, as you see fit.

Reply:
Done